lib/GameController.js

```diff
@@ -223,7 +224,9 @@ GameController.senseNearbyUnits = function(id) {
             if (loc.distanceSquaredTo(unit.location) <= unit.sensorRadiusSquared) {
                 var target = map.getUnitAtLoc(loc);
                 if (target) {
-                    result.push(target.getUnitInfo());
+                    if (enemiesOnly === undefined || (target.team != Team.NEUTRAL && target.team != unit.team)) {
```

Comment:
I'll change `enemiesOnly === undefined` to `!enemiesOnly`, then someone can pass false as a param and it will work as intended